client/src/components/EditProfile/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,33 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    input: {
+      display: 'none',
+    },
+    avatar: {
+      width: '300px',
+      height: '320px',
+    },
```

Comment:
This seems a little too large compared to the avatar in the mocks, maybe we can reduce its size a little bit?

We could also consider improving responsiveness by using a [relative unit](https://www.w3schools.com/cssref/css_units.asp) instead.